### DISCUSSION TITLE
Is there a way to use bleak without asyncio ?

### DISCUSSION THREAD

**Discussion by 0xARROWK** [hbldh/bleak#1250] [Q&A] [answered]
Hello !

I have an application without any asyncio usage and for integrating bleak into this application I will have to rewrite all functions with async/await. Is there a way to avoid to use asyncio for bleak ? If no that's okay, I ask just in case.

Thanks :)

**Comment by dlech** [comment, marked as answer]
Please see https://github.com/hbldh/bleak/issues/826.

**Comment by 0xARROWK** [comment]
Thank you for your time, I'll check it
